app.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_reader :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
```

Comment:
- [ ] When displaying the list of people, please ensure to include `Age` and the designation whether the person is a `[Student]` or `[Teacher]` as shown in the example video. You can achieve the latter using `person.class` 🙂️